src/client/api/types.ts

```diff
@@ -78,7 +78,7 @@ export type PythonApi = {
      */
     getActivatedEnvironmentVariables(
         resource: Resource,
-        interpreter?: PythonEnvironment,
+        interpreter: PythonEnvironment,
```

Comment:
This is never undefined, we always pass this around, hence changed this to not be nullable.
Makes the code cleaner